src/dispatch/case/flows.py

```diff
@@ -174,6 +174,74 @@ def create_conversation(case: Case, conversation_target: str, db_session: Sessio
     return conversation
 
 
+def create_conversation_flow(
```

Comment:
We have  a function for creating a conversation and one for adding participants to it in `conversation/flows.py`. We should refactor them to support case-specific conversations and delete `create_conversation_flow` and `create_conversation` above from this module, so we only have to maintain logic in one place.